website/docs/how-to/using-a-capacitor-plugin.md

```diff
@@ -2,3 +2,276 @@
 title: How To Use a Capacitor Plugin
 sidebar_label: Use a Capacitor Plugin
 ---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Ionic Portals uses Capacitor under the hood, meaning that you can use existing Capacitor Plugins in your Portals. These plugins allow Portals to use native functionality without much setup on either the native or web developers part.
+
+<Tabs 
+    defaultValue="ios" 
+    values={[
+        { label: 'iOS', value: 'ios', },
+        { label: 'Android', value: 'android', },
+    ]}
+>
+<TabItem value="ios">
+
+## Native Usage
+In order to use a Capacitor Plugin, you need to install the plugin as a dependency in your `Podfile`.
+
+```ruby
+# Podfile
+pod 'IonicPortals', '~> 0.2.0'
+pod 'CapacitorStorage', '~> 1.2.0'
+```
+
+:::warning
+Make sure that the versions in your `Podfile`, `build.gradle`, and `package.json` all match! Otherwise you will run into errors.
+:::
```

Comment:
Suggesting making the Admonition less scary/red and wording this piece for iOS specifically in the iOS tab.

```suggestion
:::caution
Make sure that the versions in your `Podfile` and `package.json` match! Otherwise you could run into errors.
:::
```